yellow-paper/docs/cryptography/protocol-overview.md

```diff
@@ -0,0 +1,113 @@
+- range-check gate
+- plookup table gate
+- memory-checking gates
+- non-native field arithmetic gates
+
+## Honk
+
+Honk is a variant of the PLONK protocol. Plonk performs polynomial testing via evaluating a polynomial relation is zero modulo the vanishing polynomial of a multiplicative subgroup. Honk performs the polynomial testing via evaluating, using a sumcheck protocol, that a relation over multilinear polynomials vanishes when summed over a boolean hypercube.
+
+The first protocol to combine Plonk and the sumcheck protocol was [HyperPlonk](https://eprint.iacr.org/2022/1355)
+
+Honk uses a custom arithmetisation that extends the Ultra circuit arithmetisation (not yet finalized)
+
+# Incrementally Verifiable Computation Subprotocols
+
+An Incrementally Verifiable Computation (IVC) scheme describes a protocol that enables multiple successive proofs to evolve the value taken by some defined persistent state over time.
+
+IVC schemes are used by Aztec in two capacities:
+
+1. to compute a client-side proof of one transaction execution.
+2. to compute a proof of a "rollup" circuit, that updates rollup state based on a block of user transactions
+
+Both use IVC schemes. Client-side, each function call in a transaction is a "step" in the IVC scheme. Rollup-side, aggregating two transaction proofs is a "step" in the IVC scheme.
+
+The client-side IVC scheme is substantially more complex than the rollup-side scheme due to performance requiremenmts.
+
+Rollup-side, each "step" in the IVC scheme is a Honk proof, which are recursively verified. As a result, no protoocols other than Honk are required to execute rollup-side IVC.
+
+We perform one layer of "proof-system compression" in the rollup. The final proof of block-correctness is constructed as a Honk proof. An UltraPlonk circuit is used to verify the correctness of the Honk proof, so that the proof that is verified on-chain is an UltraPlonk proof (verification gas costs are lower for UltraPlonk vs Honk).
+
+The following sections list the protocol components required to implement client-side IVC.
+
+## Protogalaxy
+
+The [Protogalaxy](https://eprint.iacr.org/2023/1106) protocol defines a folding scheme that enables instances of a relation to be folded into a single instance of a "relaxed" form of the original relation.
+
+It is a variant of [Protostar](https://eprint.iacr.org/2023/620). Unlike Protostar, Protogalaxy enables multiple instances to be efficiently folded into the same accumulator instance.
+
+The Protogalaxy protocol is split into two subprotocols, each modelled as interactive protocols between a Prover and a Verifier.
+
+#### Protogalaxy Fold
+
+The "Fold" Prover/Verifier validates that `k` instances of a defined relation (in our case the Honk relation) have been correctly folded into an accumulator instance.
+
+#### Protogalaxy Decider
+
+The "Decider" Prover/Verifier valiate whether an accumulator instance correctly satisfies the accumulator relation.
```

Comment:
```suggestion
The "Decider" Prover/Verifier validate whether an accumulator instance correctly satisfies the accumulator relation.
```

Reply:
maybe we should also say that the accumulator being satisfiable inductively shows that all instances that have been folded were satisfiable as well